search-app/src/cmr/search/services/parameters/converters/range_facet.clj

```diff
@@ -68,7 +68,8 @@
         min-value (convert (Float/parseFloat (get range-part1 0)) unit-part1)
         max-value (if range-part2
                     (convert (Float/parseFloat (get range-part2 0)) (get range-part2 1))
-                    (Float/MAX_VALUE))]
+                    ;;TODO: "pull-before-flight": send to bamboo for testing, kondo suggestion
+                    Float/MAX_VALUE)]
```

Comment:
I'm removing this comment in the next version (I assume changes will be suggested). It is here to mark what I thought was a strange clj-kondo change that I wanted tested in bamboo.